README.md

```diff
@@ -34,7 +23,10 @@ See the [API documentation](http://docs.confluent.io/current/clients/confluent-k
 Usage
 =====
 
-**Producer:**
+Below are some examples of typical usage. For more examples, including [how to configure](examples/confluent_cloud.py) the python client for use with
```

Comment:
@ybyzek will definitely want a link to https://github.com/confluentinc/examples/tree/5.3.1-post/clients/cloud/python

Reply:
i'll add a link

Reply:
I want to also add links to blog posts, and generally organize information/links out logically as per: https://www.divio.com/blog/documentation/